app/Repositories/Eloquents/RoleUserEloquentRepository.php

```diff
@@ -24,14 +24,15 @@ public function store($roles, $userId)
         }
     }
 
-    public function destroy($userId){
+    public function destroy($userId)
+    {
         $role_users = $this->model()
             ->where('user_id', $userId)
             ->get();
 
-        if($role_users) {
-            foreach ($role_users as $role_user) {
-                $role_user->delete();
+        if ($role_users) {
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Repositories/Eloquents/RoleUserEloquentRepository.php</br>Variable "role_users" is not in valid camel caps format